src/core/MenuSelect/style.ts

```diff
@@ -49,7 +49,9 @@ export const InputBaseWrapper = styled.div`
   }}
 `;
 
-export const StyledInputBase = styled(InputBase)<{ search: boolean }>`
+export const StyledInputBase = styled(InputBase, {
+  shouldForwardProp: (prop) => !doNotForwardProps.includes(prop as string),
```

Comment:
remove warning on accidentally passing `search` attr to the underlying DOM element!